src/style.css

```
@@ -1 +1,36 @@
-/* add styles here */
+
+
+body {
+  margin: 0;
+}
+
+
+
+.stars {
+  display: flex;
+}
+
+
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+}
+
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg');
+}
+
+
+.stars__star :last-child {
+  margin-right: 4px;
+}
```

```suggestion
.stars__star:not(:last-child) {
  margin-right: 4px;
}
```

Remove margin-right only from the last star in a row